_data/templates.json

```diff
@@ -1355,6 +1365,39 @@
 	},
 	"a11yGuidance": "No accessibility guidance.",
 	"variations": [
+		{
+			"name": {
+				"en": "Topic page with stacked header",
+				"fr": "Page de sujet avec en-tête superposée"
+			},
+			"status": "stable",
+			"description": {
+				"en": "Use the stacked main page title to display the page's section title as well as the page title.",
+				"fr": "Utilisez le titre de la page principale superposé pour afficher le titre de la section de la page ainsi que le titre de la page."
+			},
+			"guidance": {
+				"en": "https://design.canada.ca/mandatory-templates/theme-topic.html",
+				"fr": "https://conception.canada.ca/modeles-obligatoire/theme-sujet.html"
+			},
+			"iteration": "_:iteration_topic_4",
+			"example": [
+				{
+					"en": { "href": "topic-en.html", "text": "Topic page with stacked header" },
+					"fr": { "href": "topic-fr.html", "text": "Page de sujet avec en-tête superposée" }
```

Comment:
Can you please remove the extra "e" at the end of the following sentence: "Titre du sujet sans en-tête superposée"